src/conv.jl

```diff
@@ -202,11 +202,21 @@ for (front_name, backend, signature) in (
                                 C_in = channels_in(cdims) ÷ groupcount(cdims),
                                 C_out = channels_out(cdims) ÷ groupcount(cdims))
 
-            Threads.@sync for (xc, wc) in zip(x_cs, w_cs)
+            function do_work(xc, wc)
```

Comment:
```suggestion
            function conv_group(xc, wc)
```
For ease of debugging, I think it would help to give each `do_work` function a unique name. Ideally a more informative one too.